microprot/scripts/split_search.py

```diff
@@ -577,9 +628,9 @@ def pretty_output(mask_out):
 @click.option('--subseq_fp', '-o', default=None,
               type=click.Path(),
```

Comment:
This `type` doesn't include any parameters. Are you sure? Does this file have to be (non)existed, (non)readable, (non)writable, path/file, etc.?